### PR TITLE
[SAI-PTF] Add SAI-PTF README and docs

### PR DESCRIPTION
**Pull request by richardyu-ms**
Add SAI-PTF README and docs for how to set up the SAI-PTF testing environment.

includes the sections about:
- Introduction about the structure and topology
- How to set up the testbed
- Examples for running the tests

For how to set up the, it includes two approaches
- automated approach, setup the test bed leverage by sonic-buildimage and sonic-mgmt
- Manual approach, build install the packages step by step

### CODE REVIEW

**Comment by richardyu-ms** [comment]
@reshmaintel could you plesae help make a review.
thanks.

**Review by reshmaintel — APPROVED** [review]

**Comment by reshmaintel** [comment]
@vmytnykx please review as well, thanks.

File: ptf/docs/ExamplePhysicalConnection.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+This example follows the [Testbed Setting (the third config)](https://github.com/sonic-net/sonic-mgmt/blob/master/ansible/testbed.csv#L4) and [Connection Graph](https://github.com/sonic-net/sonic-mgmt/commit/005d792675bbd21184534089d41e2be7e60c2735).
+
+## Key Components
+The physical connection of the testbed is designed like below:
+
+![Physical topology](./img/physical_connection.png)
+
+Key components in the physical connection:
+
+- Test servers
```
Comment (Gfrom2016) [inline ptf/docs/ExamplePhysicalConnection.md:19]: redundant line?

File: ptf/docs/PTF-SAIv2Overview.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+ > For more introduction about how to use the sonic-buildimage, please refer to [sonic-buildimage user guide](https://github.com/Azure/sonic-buildimage)
+
+In the following, we use other SONiC scripts to help setup the [SAI PTF topology](https://github.com/sonic-net/sonic-mgmt/blob/master/docs/testbed/README.testbed.Overview.md#ptf-type-topology) environment with all the testing components mentioned in [SAI PTF introduction and manually setup Testbed](ManuallySetupTestbedGuide.md).
+
+### Build PTF-SAIv2 components
+*In this section, we will build the test-related resources.*
+
+For SAI-PTFv2 test, we need to build components:
+1. Docker PTF, which contains all the runtime dependencies
+2. Docker SAIServerv2, which contains the RPC Server (as described in ##Instructuon section), SAI SDK, and all the running dependences
```
Comment (Gfrom2016) [inline ptf/docs/PTF-SAIv2Overview.md:69]: Reference for Instruction section didn't work here

**Review by Gfrom2016 — APPROVED** [review]

**Pull request merged**